gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -1,5 +1,71 @@
 """Script to create release sites HT for v4 genomes."""
 import hail as hl
+from gnomad.utils.vep import filter_vep_transcript_csqs
+
+# TODO: steps to create release sites HT
+# input: VRS-annotated sites HT v3.1.4
+# move SIFT and PolyPhen to insilico_predictors in vep_ht
+# remove missing fields in vep_ht
+# drop vep from site HT and merge with vep_ht
+# remove old insilico predictors and replace with new ones (the left predictors)
+# update dbSNP to dbSNP 156
+# update freq, freq_meta, freq_index_dict by integrating updates in HGDP/TGP
+# rerun inbreeding_coefficient with the callstats from the new freq
+# replace oth with remaining in global fields
+# update global fields
+
+
+def get_sift_polyphen_from_vep(ht: hl.Table) -> hl.Table:
+    """
+    Get SIFT and PolyPhen scores from VEP 105 annotations.
+
+    .. note::
+     This function is to get a max of SIFT and PolyPhen scores from mane_select
+     transcripts, otherwise get a max of SIFT and PolyPhen scores from canonical
+     transcripts. It will also drop the SIFT and PolyPhen scores and predictions
+     from the transcript_consequences struct.
+
+    :param ht: VEP105 annotated Hail Table.
+    :return: Hail Table with VEP105 annotations and  SIFT and PolyPhen scores
+    extracted and stored in insilico_predictors struct.
```

Comment:
```suggestion
    :return: Hail Table with SIFT and PolyPhen scores
```
I'm  think this should just return the scores but curious what your thoughts are on returning the rest of the table? We'll need to use this for v4 exomes as well so it should be pretty general and not built into the v4 genomes pipeline.

Reply:
I think this should also be part of adding VEP annotations to the VRS input.

Reply:
Maybe we just make one function for both?